csm/core/agent/csm_agent.py

```diff
@@ -44,10 +44,13 @@ def init():
         file_size_in_mb = Conf.get(const.CSM_GLOBAL_INDEX, "Log>file_size")
         log_level = "DEBUG" if Options.debug else Conf.get(const.CSM_GLOBAL_INDEX, "Log>log_level")
         console_output = Conf.get(const.CSM_GLOBAL_INDEX, "Log>console_logging") == "true"
+        csm_log_path = Conf.get(const.CSM_GLOBAL_INDEX, "Log>log_path")
+        machine_id = Conf.machine_id
```

Comment:
Check Null for machine ID and add Warn log and do not append the path

Reply:
???

Reply:
Pushing the change.. 
I can not add warning logs as we are initializing logs after this